### PR TITLE
Render (some) BFME FXParticleSystems

### PR DESCRIPTION
**Pull request by tgjones**

### CODE REVIEW

**Comment by tgjones** [comment]
This still crashes on many, probably most, FXParticleSystems in BFME I. But it works for the very first one (AmorySparks). This can be followed up by separate work to fix the other crashes.

File: src/OpenSage.Game/Graphics/ParticleSystems/ParticleSystem.cs [hunk 1]
```diff
@@ -53,13 +51,13 @@ public sealed class ParticleSystem : DisposableBase
...
 
-        public ParticleSystemDefinition Definition { get; }
+        public FXParticleSystemTemplate Definition { get; }
 
         public ParticleSystemState State { get; private set; }
 
         public ParticleSystem(
             ContentManager contentManager,
-            ParticleSystemDefinition definition,
+            FXParticleSystemTemplate definition,
```
[the remark this answers is not in the document]
Comment (tgjones) [reply src/OpenSage.Game/Graphics/ParticleSystems/ParticleSystem.cs]: Done. Does this look okay now?

[the remark this answers is not in the document]
Comment (Tarcontar) [reply src/OpenSage.Game/Graphics/ParticleSystems/ParticleSystem.cs]: Looks good now.

[the remark this answers is not in the document]
Comment (tgjones) [reply src/OpenSage.Game/Graphics/ParticleSystems/ParticleSystem.cs]: Thanks!

**Pull request merged**